corral/test/util.pony

```diff
@@ -39,14 +39,30 @@ class val DataClone
   let _root: FilePath
   let _dir: FilePath
 
-  new val create(h: TestHelper, subdir: String = "") ? =>
+  new val create(h: TestHelper, subdirs: (Array[String] val | String | None) = None) ? =>
     let auth = h.env.root as AmbientAuth
     let src_root = FilePath(auth, "corral/test/testdata")?
 
     _root = FilePath.mkdtemp(auth, "test_scratch.")?
-    _dir = _root.join(subdir)?
 
-    Copy.tree(src_root, _root, subdir)?
+    let subdirs': Array[String] val =
+      match subdirs
```

Comment:
I'd suggest using (consume `subdirs`) to avoid possible problems with using `subdirs` instead of `subdirs'` later in the function.

Reply:
Done.